packages/generators/templates/component/{{name}}.{{mdx}}

```diff
@@ -24,18 +20,13 @@ import { {{ name }} } from "@jobber/components/{{ name }}";
 </Playground>
 
 <!--
-  It is not necessary to include Design & Usage Guidlelines. Use this section
-  to answer common questions and considerations about component usage.
+  INTERFACE:
+  Provide an example of what the component looks like in code. How would you use
+  it from another React component? This should consist primarily of code blocks.
+  This section can be deleted once the component is built, when the Playground
+  will demonstrate this at a higher fidelity.
```

Comment:
Do we want to say that this is mainly for the proposal stage of the component development?